src/config.cpp

```diff
@@ -311,6 +311,9 @@ void parse_program_options(int argc, const char** argv)
                 ->value_name("N")
                 ->default_value(11010113),
             "Sampling period (in number of events specified by -e).")
+        ("enable-system-sampling",
```

Comment:
There is no other option with the prefix enable, so I'd prefer `--system-sampling` with the short option `-A`